### DISCUSSION TITLE
[General] Get the build version of windows on inventory

### DISCUSSION THREAD

**Discussion by xillibit** [glpi-project/glpi-agent#408] [General]
### Your question

Hello,

When the inventory is done by the agent, is-it possible to get the build version of windows under `HKEY_LOCAL_MACHINE/Software/Microsoft/Windows NT/CurrentVersion/` : 

![image](https://github.com/glpi-project/glpi-agent/assets/876395/61958e0f-4a66-4b38-93c6-3c72bfcd5304)

**Comment by g-bougard** [comment]
Hi @xillibit 
this is still the case in nightly version.
Get it from there: https://nightly.glpi-project.org/glpi-agent/
You'll see the UBR in the service pack field in GLPI

**Comment by xillibit** [comment]
Hello,

I'am using the GLPI agent in version 1.4 so i didn't know that this thing was added in 1.5 nightly build, i will give a try

**Comment by xillibit** [comment]
Hello,

I have tried with `GLPI-Agent v1.5-git4f85e3be nightly build` i have seen an error in the log : 

```
[Tue Jun 20 09:38:26 2023][info] New inventory from ads-n.MYDOMAIN.LOCAL-2023-06-20-09-38-21 for server0
[Tue Jun 20 09:39:14 2023][info] running task Collect
[Tue Jun 20 09:39:14 2023][error] [http client] Can't decode JSON content, starting with: 
         

<!DOCTYPE html>
<html lang="fr" >
<head>
   <title>Authentification - GLPI</title>

   <meta charset="utf-8" />
...
[Tue Jun 20 09:39:14 2023][info] Collect task not supported by server0
```

The first time it has make the inventory, i have checked in GLPI the computer right here and in the details of computer i can see it was the agent in 1.5 version. Then i needed to renamme the computer and i launched on the computer an inventory but the computer hasn’t appear in GLPI.

I have uninstalled the agent in 1.5 and put back the 1.4 version

**Reply by g-bougard** [reply]
This error is related to Collect task, not inventory one. So don't worry about this error. If you don't need this task, just use [no-task configuration](https://glpi-agent.readthedocs.io/en/latest/configuration.html#no-task) to set `no-task = collect`.
If you rename the computer, you may need to delete the `var/Target.dump` file to reset the agent name.
